### DISCUSSION TITLE
🎉 Geany writeup in The Register 🎉

### DISCUSSION THREAD

**Discussion by rdipardo** [geany/geany#3712] [Show and tell]
Today [The Register](https://www.theregister.com) published a small profile of the Geany project to mark the 2.0 release: ["Veteran editors Notepad++ and Geany hit milestone versions"](https://www.theregister.com/2023/12/06/veteran_editors_notepad_and_geany)

**Comment by andy5995** [comment]
Thanks @rdipardo - I shared this with both my nephews, who are still using Geany after I introduced them to it many years ago.
